code/modules/admin/verbs/pray.dm

```diff
@@ -31,6 +31,11 @@
 		usr.whisper(orig_message)
 	to_chat(usr, "Your prayers have been received by the gods.")
 
+	for(var/datum/religion/R in ticker.religions)
+		if(findtext(lowertext(R.deity_name), "buttbot") || findtext(lowertext(R.deity_name), "butt bot"))
```

Comment:
Rather than doing a deity name comparison, don't you think it would be more OOP to create a buttbot religion (with both of these as key options)? Plus that would improve the framework of your buttbot joke religion so that people could add stuff like giving the chaplain a free ass hat, or maybe some day someone will make a butt bible sprite.

Reply:
Yes, but I didn't want to come up with a whole religion for it

Reply:
It's really simple, here, I'll give you code for it

```
/datum/religion/buttbot
	name = "Buttbot"
	deity_name = "Buttbot"
	bible_name = "The Buttbot Radio Logs"
	male_adept = "Roboticist"
	female_adept = "Roboticist"
	keys = list("buttbot", "butt bot", "butt")
```

If someone wants to expand on it with other features like a hat or conversion ceremony, they can do that on their own.